src/os_mswin.c

```diff
@@ -72,6 +72,46 @@
 # endif
 #endif
 
+# ifndef IO_REPARSE_TAG_APPEXECLINK
+#  define IO_REPARSE_TAG_APPEXECLINK 0x8000001B
+# endif
```

Comment:
Other `IO_REPARSE_TAG_...` definitions exist in `os_win32.h`.
Is it better to move it there?

Reply:
That's definitely better, I somehow missed the whole `.h` file...